packages/@types/function-tree/Path.d.ts

```diff
@@ -0,0 +1,14 @@
+import { Payload } from "./primitives";
+
+// Is 'path' correct ?
+declare interface IPath {
```

Comment:
Not exactly sure what you mean by "correct" :-) But yeah, path is the prop that holds the name of the path